haiku-games/hexvexed/hexvexed-0.0.1.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="A tile puzzle game for Haiku"
+DESCRIPTION="HexVexed is a maddening addictive and easy to play puzzle game.\
```

Comment:
You should either drop the backslash in the end of this line, or add a white space before it, right after "game.", otherwise the description in HaikuDepot will show "game.Arrange" (instead of "game. Arrange").
I guess that adding the missing white space is better than dropping the backslash.